lib/tiki/torch/publishing/retries.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module Tiki
+  module Torch
+    module Publishing
+      class Retries
+        class Entry
+          attr_reader :topic_name, :event
+
+          # @param [String] topic_name the name of the topic to retry
+          # @param [Object] event the event to retry
+          def initialize(topic_name, event)
+            @topic_name = topic_name
+            @event      = event
+            @tries      = 1
+          end
+
+          # @return [String] A unique identifier for the topic_name + event
+          def fingerprint
+            @fingerprint ||= format '%s:%s',
+                                    @topic_name.to_s,
+                                    (@event.respond_to?(:fingerprint) ? @event.fingerprint : @event.to_s)
+          end
+
+          # Increase the quantity of tries for the entry
```

Comment:
You may want to comment that the try counter is currently informational-only.

Reply:
pls check out the new documentation